test/linters/linters.test.js

```diff
@@ -10,6 +10,7 @@ const golintParams = require("./params/golint");
 const prettierParams = require("./params/prettier");
 const stylelintParams = require("./params/stylelint");
 const swiftlintParams = require("./params/swiftlint");
+const ruboCopParams = require("./params/rubo-cop");
```

Comment:
Could you please rename this file to `rubocop.js` (since there's no space in the name)?